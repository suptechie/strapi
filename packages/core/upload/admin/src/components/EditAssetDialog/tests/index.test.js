/**
 *
 * Tests for EditAssetDialog
 *
 */

import React from 'react';
import { render } from '@testing-library/react';
import { ThemeProvider, lightTheme } from '@strapi/parts';
import { IntlProvider } from 'react-intl';
import { EditAssetDialog } from '../index';

const messages = {
  'upload.component.name': 'Edit Asset Dialog',
};

describe('<EditAssetDialog />', () => {
  it('renders and matches the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <ThemeProvider theme={lightTheme}>
        <IntlProvider locale="en" messages={messages} defaultLocale="en">
          <EditAssetDialog />
        </IntlProvider>
      </ThemeProvider>
    );

    expect(firstChild).toMatchInlineSnapshot(`
      .c0 {
        border: 0;
        -webkit-clip: rect(0 0 0 0);
        clip: rect(0 0 0 0);
        height: 1px;
        margin: -1px;
        overflow: hidden;
        padding: 0;
        position: absolute;
        width: 1px;
      }

      <div
        class="c0"
      >
        <p
          aria-live="polite"
          aria-relevant="all"
          id="live-region-log"
          role="log"
        />
        <p
          aria-live="polite"
          aria-relevant="all"
          id="live-region-status"
          role="status"
        />
        <p
          aria-live="assertive"
          aria-relevant="all"
          id="live-region-alert"
          role="alert"
        />
      </div>
    `);
  });
});
