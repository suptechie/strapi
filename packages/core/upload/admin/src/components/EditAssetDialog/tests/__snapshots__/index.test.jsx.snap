// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditAssetDialog /> renders and matches the snapshot 1`] = `
.c0 {
  margin-inline-start: -250px;
  position: fixed;
  left: 50%;
  top: 4.6rem;
  z-index: 10;
  width: 50rem;
}

.c6 {
  padding-inline-end: 40px;
  padding-inline-start: 40px;
  position: fixed;
  z-index: 4;
}

.c9 {
  background: #ffffff;
  border-radius: 4px;
  box-shadow: 0px 2px 15px rgba(33, 33, 52, 0.1);
  width: 83rem;
}

.c10 {
  background: #f6f6f9;
  padding-block-start: 16px;
  padding-inline-end: 20px;
  padding-block-end: 16px;
  padding-inline-start: 20px;
}

.c14 {
  background: #ffffff;
  padding-block: 8px;
  padding-inline: 8px;
  border-radius: 4px;
  border-color: #dcdce4;
  border: 1px solid #dcdce4;
  cursor: pointer;
}

.c17 {
  padding-block: 32px;
  padding-inline: 32px;
}

.c21 {
  background: #eaeaef;
  border-radius: 4px;
  border-color: #dcdce4;
  border: 1px solid #dcdce4;
}

.c23 {
  padding-inline-end: 12px;
  padding-inline-start: 12px;
}

.c3 {
  background: #212134;
  padding-block: 8px;
  padding-inline: 8px;
  border-radius: 4px;
  position: absolute;
}

.c28 {
  padding-inline-end: 8px;
  padding-inline-start: 8px;
}

.c30 {
  background: #f6f6f9;
  padding-block-start: 16px;
  padding-inline-end: 24px;
  padding-block-end: 16px;
  padding-inline-start: 24px;
  border-radius: 4px;
}

.c34 {
  display: flex;
}

.c40 {
  padding-inline-end: 12px;
}

.c45 {
  background: #4945ff;
  padding-block: 8px;
  padding-inline: 8px;
  padding-inline-end: 16px;
  padding-inline-start: 16px;
  border-radius: 4px;
  border-color: #4945ff;
  border: 1px solid #4945ff;
  cursor: pointer;
}

.c13 {
  font-size: 1.4rem;
  line-height: 1.43;
  font-weight: 600;
  color: #32324d;
}

.c5 {
  font-size: 1.2rem;
  line-height: 1.33;
  font-weight: 600;
  color: #ffffff;
}

.c32 {
  font-weight: 600;
  font-size: 1.1rem;
  line-height: 1.45;
  text-transform: uppercase;
  color: #666687;
}

.c33 {
  font-size: 1.2rem;
  line-height: 1.33;
  color: #4a4a6a;
}

.c35 {
  font-size: 1.2rem;
  line-height: 1.33;
  font-weight: 600;
  color: #32324d;
}

.c39 {
  font-size: 1.2rem;
  line-height: 1.33;
  color: #666687;
}

.c47 {
  font-size: 1.2rem;
  line-height: 1.33;
  font-weight: 600;
  color: currentcolor;
}

.c1 {
  align-items: stretch;
  display: flex;
  flex-direction: column;
  gap: 8px;
}

.c7 {
  align-items: center;
  display: flex;
  flex-direction: row;
  justify-content: center;
}

.c12 {
  align-items: center;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
}

.c24 {
  align-items: center;
  display: flex;
  flex-direction: row;
  justify-content: flex-end;
}

.c26 {
  align-items: center;
  display: flex;
  flex-direction: row;
  gap: 4px;
}

.c29 {
  align-items: stretch;
  display: flex;
  flex-direction: column;
  gap: 12px;
}

.c31 {
  align-items: stretch;
  display: flex;
  flex-direction: column;
  gap: 4px;
}

.c36 {
  align-items: center;
  display: flex;
  flex-direction: row;
  gap: 8px;
  justify-content: space-between;
}

.c44 {
  align-items: center;
  display: flex;
  flex-direction: row;
  gap: 8px;
}

.c2 {
  border: 0;
  clip: rect(0 0 0 0);
  height: 1px;
  margin: -1px;
  overflow: hidden;
  padding: 0;
  position: absolute;
  width: 1px;
}

.c15 {
  position: relative;
  outline: none;
}

.c15[aria-disabled='true'] {
  pointer-events: none;
}

.c15:after {
  transition-property: all;
  transition-duration: 0.2s;
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -4px;
  bottom: -4px;
  left: -4px;
  right: -4px;
  border: 2px solid transparent;
}

.c15:focus-visible {
  outline: none;
}

.c15:focus-visible:after {
  border-radius: 8px;
  content: '';
  position: absolute;
  top: -5px;
  bottom: -5px;
  left: -5px;
  right: -5px;
  border: 2px solid #4945ff;
}

.c46 {
  height: 3.2rem;
  border: 1px solid #dcdce4;
  background: #ffffff;
  color: #32324d;
}

.c46 svg {
  height: 1.2rem;
}

.c46[aria-disabled='true'] {
  border: 1px solid #dcdce4;
  background: #eaeaef;
  color: #666687;
}

.c46[aria-disabled='true']:active {
  border: 1px solid #dcdce4;
  background: #eaeaef;
  color: #666687;
}

.c46:hover {
  background-color: #f6f6f9;
}

.c46:active {
  background-color: #eaeaef;
}

.c48 {
  height: 3.2rem;
  border: 1px solid #d9d8ff;
  background: #f0f0ff;
  color: #271fe0;
}

.c48 svg {
  height: 1.2rem;
}

.c48[aria-disabled='true'] {
  border: 1px solid #dcdce4;
  background: #eaeaef;
  color: #666687;
}

.c48[aria-disabled='true']:active {
  border: 1px solid #dcdce4;
  background: #eaeaef;
  color: #666687;
}

.c48:hover {
  background-color: #ffffff;
}

.c48:active {
  background-color: #ffffff;
  border: 1px solid #4945ff;
  color: #4945ff;
}

.c49 {
  height: 3.2rem;
  color: #ffffff;
}

.c49 svg {
  height: 1.2rem;
}

.c49[aria-disabled='true'] {
  border: 1px solid #dcdce4;
  background: #eaeaef;
  color: #666687;
}

.c49[aria-disabled='true']:active {
  border: 1px solid #dcdce4;
  background: #eaeaef;
  color: #666687;
}

.c49:hover {
  border: 1px solid #7b79ff;
  background: #7b79ff;
}

.c49:active {
  border: 1px solid #4945ff;
  background: #4945ff;
}

.c4 {
  z-index: 4;
  display: none;
}

.c38 {
  border: none;
  border-radius: 4px;
  padding: 0;
  padding-bottom: 0.8rem;
  padding-top: 0.8rem;
  color: #32324d;
  font-weight: 400;
  font-size: 1.4rem;
  line-height: 2.2rem;
  display: block;
  width: 100%;
  background: inherit;
}

.c38 ::placeholder {
  color: #8e8ea9;
  opacity: 1;
}

.c38[aria-disabled='true'] {
  color: inherit;
}

.c38:focus {
  outline: none;
  box-shadow: none;
}

.c37 {
  border: 1px solid #dcdce4;
  border-radius: 4px;
  background: #ffffff;
  outline: none;
  box-shadow: 0;
  transition-property: border-color,box-shadow,fill;
  transition-duration: 0.2s;
}

.c37:focus-within {
  border: 1px solid #4945ff;
  box-shadow: #4945ff 0px 0px 0px 2px;
}

.c19 {
  display: grid;
  grid-template-columns: repeat(12, 1fr);
  gap: 16px;
}

.c20 {
  grid-column: span 6;
  max-width: 100%;
}

.c16 {
  border-color: #dcdce4;
  height: 3.2rem;
  width: 3.2rem;
  color: #8e8ea9;
}

.c16:hover,
.c16:focus {
  color: #666687;
}

.c16[aria-disabled='true'] {
  color: #666687;
}

.c8 {
  background: #32324d1F;
  inset: 0;
}

.c11 {
  border-radius: 4px 4px 0 0;
  border-bottom: 1px solid #eaeaef;
}

.c43 {
  border-radius: 0 0 4px 4px;
  border-top: 1px solid #eaeaef;
}

.c18 {
  overflow: auto;
  max-height: 60vh;
}

.c41 {
  background: transparent;
  border: none;
  position: relative;
  z-index: 1;
}

.c41 svg {
  height: 1.1rem;
  width: 1.1rem;
}

.c41 svg path {
  fill: #666687;
}

.c42 {
  display: flex;
  background: none;
  border: none;
}

.c42 svg {
  width: 0.9rem;
}

.c22 {
  position: relative;
}

.c27 {
  position: relative;
  text-align: center;
  background: repeating-conic-gradient(       #f6f6f9 0% 25%,       transparent 0% 50%     ) 50%/20px 20px;
}

.c27 svg {
  font-size: 4.8rem;
  height: 26.4rem;
}

.c27 img,
.c27 video {
  margin: 0;
  padding: 0;
  max-height: 26.4rem;
  max-width: 100%;
}

.c25 {
  height: 5.2rem;
}

@media (max-width: 1100px) {
  .c20 {
    grid-column: span;
  }
}

@media (max-width: 550px) {
  .c20 {
    grid-column: span 12;
  }
}

<body
  class="lock-body-scroll"
>
  <div>
    <div
      class="c0 c1"
    />
    <span
      class="c2"
    >
      <p
        aria-live="polite"
        aria-relevant="all"
        id="live-region-log"
        role="log"
      />
      <p
        aria-live="polite"
        aria-relevant="all"
        id="live-region-status"
        role="status"
      />
      <p
        aria-live="assertive"
        aria-relevant="all"
        id="live-region-alert"
        role="alert"
      />
    </span>
  </div>
  <div
    class=""
  >
    <div
      class="c3 c4"
      id=":r0:"
      role="tooltip"
    >
      <p
        class="c5"
      >
        Delete
      </p>
    </div>
  </div>
  <div
    class=""
  >
    <div
      class="c3 c4"
      id=":r2:"
      role="tooltip"
    >
      <p
        class="c5"
      >
        Download
      </p>
    </div>
  </div>
  <div
    class=""
  >
    <div
      class="c3 c4"
      id=":r4:"
      role="tooltip"
    >
      <p
        class="c5"
      >
        Copy link
      </p>
    </div>
  </div>
  <div
    class=""
  >
    <div
      class="c3 c4"
      id=":r6:"
      role="tooltip"
    >
      <p
        class="c5"
      >
        Crop
      </p>
    </div>
  </div>
  <div
    class=""
  >
    <div
      class="c6 c7 c8"
    >
      <div>
        <div>
          <div
            aria-labelledby="title"
            aria-modal="true"
            class="c9"
            role="dialog"
          >
            <div
              class="c10 c11"
            >
              <div
                class="c12"
              >
                <h2
                  class="c13"
                  id="title"
                >
                  Details
                </h2>
                <button
                  aria-disabled="false"
                  class="c14 c7 c15 c16"
                  type="button"
                >
                  <span
                    class="c2"
                  >
                    Close the modal
                  </span>
                  <svg
                    aria-hidden="true"
                    fill="currentColor"
                    focusable="false"
                    height="16"
                    viewBox="0 0 32 32"
                    width="16"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M26.061 23.939a1.503 1.503 0 0 1-2.125 2.125L16 18.125l-7.939 7.936a1.503 1.503 0 1 1-2.125-2.125L13.875 16 5.939 8.061a1.503 1.503 0 1 1 2.125-2.125L16 13.875l7.939-7.94a1.502 1.502 0 1 1 2.125 2.125L18.125 16z"
                    />
                  </svg>
                </button>
              </div>
            </div>
            <div
              class="c17 c18"
            >
              <div
                class="c19"
              >
                <div
                  class="c20"
                >
                  <div
                    class="c21 c22"
                  >
                    <div
                      class="c23 c24 c25"
                    >
                      <div
                        class="c26"
                      >
                        <span>
                          <button
                            aria-disabled="false"
                            aria-labelledby=":r0:"
                            class="c14 c7 c15 c16"
                            tabindex="0"
                            type="button"
                          >
                            <span
                              class="c2"
                            >
                              Delete
                            </span>
                            <svg
                              aria-hidden="true"
                              fill="currentColor"
                              focusable="false"
                              height="16"
                              viewBox="0 0 32 32"
                              width="16"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M27 6h-5V5a3 3 0 0 0-3-3h-6a3 3 0 0 0-3 3v1H5a1 1 0 0 0 0 2h1v18a2 2 0 0 0 2 2h16a2 2 0 0 0 2-2V8h1a1 1 0 1 0 0-2M14 21a1 1 0 0 1-2 0v-8a1 1 0 0 1 2 0zm6 0a1 1 0 0 1-2 0v-8a1 1 0 0 1 2 0zm0-15h-8V5a1 1 0 0 1 1-1h6a1 1 0 0 1 1 1z"
                              />
                            </svg>
                          </button>
                        </span>
                        <span>
                          <button
                            aria-disabled="false"
                            aria-labelledby=":r2:"
                            class="c14 c7 c15 c16"
                            tabindex="0"
                            type="button"
                          >
                            <span
                              class="c2"
                            >
                              Download
                            </span>
                            <svg
                              aria-hidden="true"
                              fill="currentColor"
                              focusable="false"
                              height="16"
                              viewBox="0 0 32 32"
                              width="16"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M28.5 19v7a2.5 2.5 0 0 1-2.5 2.5H6A2.5 2.5 0 0 1 3.5 26v-7a1.5 1.5 0 0 1 3 0v6.5h19V19a1.5 1.5 0 1 1 3 0m-13.561 1.061a1.5 1.5 0 0 0 2.125 0l5-5a1.502 1.502 0 1 0-2.125-2.125L17.5 15.375V5a1.5 1.5 0 1 0-3 0v10.375l-2.439-2.436a1.502 1.502 0 1 0-2.125 2.125z"
                              />
                            </svg>
                          </button>
                        </span>
                        <span>
                          <button
                            aria-disabled="false"
                            aria-labelledby=":r4:"
                            class="c14 c7 c15 c16"
                            tabindex="0"
                            type="button"
                          >
                            <span
                              class="c2"
                            >
                              Copy link
                            </span>
                            <svg
                              aria-hidden="true"
                              fill="currentColor"
                              focusable="false"
                              height="16"
                              viewBox="0 0 32 32"
                              width="16"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M17.046 23.441a1.5 1.5 0 0 1 0 2.125l-.742.743a7.502 7.502 0 1 1-10.61-10.61l3.015-3.014A7.5 7.5 0 0 1 19 12.375a1.506 1.506 0 0 1-2 2.25 4.5 4.5 0 0 0-6.171.184l-3.013 3.01a4.5 4.5 0 0 0 6.365 6.365l.743-.743a1.5 1.5 0 0 1 2.122 0m9.26-17.75a7.51 7.51 0 0 0-10.61 0l-.742.743a1.503 1.503 0 1 0 2.125 2.125l.742-.743a4.5 4.5 0 0 1 6.365 6.365l-3.014 3.015a4.5 4.5 0 0 1-6.172.179 1.506 1.506 0 1 0-2 2.25 7.5 7.5 0 0 0 10.288-.304l3.014-3.014a7.51 7.51 0 0 0 .004-10.613z"
                              />
                            </svg>
                          </button>
                        </span>
                        <span>
                          <button
                            aria-disabled="false"
                            aria-labelledby=":r6:"
                            class="c14 c7 c15 c16"
                            tabindex="0"
                            type="button"
                          >
                            <span
                              class="c2"
                            >
                              Crop
                            </span>
                            <svg
                              aria-hidden="true"
                              fill="currentColor"
                              focusable="false"
                              height="16"
                              viewBox="0 0 32 32"
                              width="16"
                              xmlns="http://www.w3.org/2000/svg"
                            >
                              <path
                                d="M30.5 24a1.5 1.5 0 0 1-1.5 1.5h-3.5V29a1.5 1.5 0 1 1-3 0v-3.5H8A1.5 1.5 0 0 1 6.5 24V9.5H3a1.5 1.5 0 0 1 0-3h3.5V3a1.5 1.5 0 0 1 3 0v19.5H29a1.5 1.5 0 0 1 1.5 1.5M13 9.5h9.5V19a1.5 1.5 0 1 0 3 0V8A1.5 1.5 0 0 0 24 6.5H13a1.5 1.5 0 0 0 0 3"
                              />
                            </svg>
                          </button>
                        </span>
                      </div>
                    </div>
                    <div
                      class="c27"
                    >
                      <img
                        alt="Screenshot 2.png"
                        src="http://localhost:1337/uploads/thumbnail_Screenshot_2_5d4a574d61.png"
                      />
                    </div>
                    <div
                      class="c28 c24 c25"
                    />
                  </div>
                </div>
                <div
                  class="c20"
                >
                  <form
                    action="#"
                    novalidate=""
                  >
                    <div
                      class="c29"
                    >
                      <div
                        class="c30"
                      >
                        <div
                          class="c19"
                        >
                          <div
                            class="c20"
                          >
                            <div
                              class="c31"
                            >
                              <span
                                class="c32"
                              >
                                Size
                              </span>
                              <span
                                class="c33"
                              >
                                102KB
                              </span>
                            </div>
                          </div>
                          <div
                            class="c20"
                          >
                            <div
                              class="c31"
                            >
                              <span
                                class="c32"
                              >
                                Dimensions
                              </span>
                              <span
                                class="c33"
                              >
                                1476✕780
                              </span>
                            </div>
                          </div>
                          <div
                            class="c20"
                          >
                            <div
                              class="c31"
                            >
                              <span
                                class="c32"
                              >
                                Date
                              </span>
                              <span
                                class="c33"
                              >
                                10/4/2021
                              </span>
                            </div>
                          </div>
                          <div
                            class="c20"
                          >
                            <div
                              class="c31"
                            >
                              <span
                                class="c32"
                              >
                                Extension
                              </span>
                              <span
                                class="c33"
                              >
                                png
                              </span>
                            </div>
                          </div>
                          <div
                            class="c20"
                          >
                            <div
                              class="c31"
                            >
                              <span
                                class="c32"
                              >
                                Asset ID
                              </span>
                              <span
                                class="c33"
                              >
                                8
                              </span>
                            </div>
                          </div>
                        </div>
                      </div>
                      <div
                        class="c31"
                      >
                        <label
                          alignitems="center"
                          class="c34 c35"
                          for=":r8:"
                          id=":r8:-label"
                        >
                          File name
                        </label>
                        <div
                          class="c23 c36 c37"
                        >
                          <input
                            aria-disabled="false"
                            class="c38"
                            id=":r8:"
                            name="name"
                            value="Screenshot 2.png"
                          />
                        </div>
                      </div>
                      <div
                        class="c31"
                      >
                        <label
                          alignitems="center"
                          class="c34 c35"
                          for=":r9:"
                          id=":r9:-label"
                        >
                          Alternative text
                        </label>
                        <div
                          class="c23 c36 c37"
                        >
                          <input
                            aria-describedby=":r9:-hint"
                            aria-disabled="false"
                            class="c38"
                            id=":r9:"
                            name="alternativeText"
                          />
                        </div>
                        <p
                          class="c39"
                          id=":r9:-hint"
                        >
                          This text will be displayed if the asset can’t be shown.
                        </p>
                      </div>
                      <div
                        class="c31"
                      >
                        <label
                          alignitems="center"
                          class="c34 c35"
                          for=":ra:"
                          id=":ra:-label"
                        >
                          Caption
                        </label>
                        <div
                          class="c23 c36 c37"
                        >
                          <input
                            aria-disabled="false"
                            class="c38"
                            id=":ra:"
                            name="caption"
                          />
                        </div>
                      </div>
                      <div
                        class="c31"
                      >
                        <div
                          class="c31"
                        >
                          <label
                            alignitems="center"
                            class="c34 c35"
                            for="asset-folder"
                            id="asset-folder-label"
                          >
                            Location
                          </label>
                          <div
                            class=" css-1iyvlxg-container"
                          >
                            <span
                              class="css-1f43avz-a11yText-A11yText"
                              id="react-select-2-live-region"
                            />
                            <span
                              aria-atomic="false"
                              aria-live="polite"
                              aria-relevant="additions text"
                              class="css-1f43avz-a11yText-A11yText"
                              role="log"
                            />
                            <div
                              class=" css-oe0cnm-control"
                            >
                              <div
                                class=" css-onz4xi-ValueContainer"
                              >
                                <div
                                  class=" css-1ryi7up-singleValue"
                                >
                                  Media Library
                                </div>
                                <input
                                  aria-activedescendant=""
                                  aria-autocomplete="list"
                                  aria-expanded="false"
                                  aria-haspopup="true"
                                  aria-invalid="false"
                                  aria-readonly="true"
                                  class="css-mohuvp-dummyInput-DummyInput"
                                  id="asset-folder"
                                  inputmode="none"
                                  role="combobox"
                                  tabindex="0"
                                  value=""
                                />
                              </div>
                              <div
                                class=" css-1hm8yhz-IndicatorsContainer"
                              >
                                <div
                                  aria-hidden="true"
                                  class="c40 c41 c42"
                                >
                                  <svg
                                    fill="currentColor"
                                    height="16"
                                    viewBox="0 0 32 32"
                                    width="16"
                                    xmlns="http://www.w3.org/2000/svg"
                                  >
                                    <path
                                      d="m26.708 12.708-10 10a1 1 0 0 1-1.415 0l-10-10A1 1 0 0 1 6 11h20a1 1 0 0 1 .707 1.707"
                                    />
                                  </svg>
                                </div>
                              </div>
                            </div>
                            <input
                              name="parent"
                              type="hidden"
                              value=""
                            />
                          </div>
                        </div>
                      </div>
                    </div>
                    <span
                      class="c2"
                    >
                      <button
                        tabindex="-1"
                        type="submit"
                      >
                        Submit
                      </button>
                    </span>
                  </form>
                </div>
              </div>
            </div>
            <div
              class="c10 c43"
            >
              <div
                class="c12"
              >
                <div
                  class="c44"
                >
                  <button
                    aria-disabled="false"
                    class="c45 c44 c15 c46"
                    type="button"
                  >
                    <span
                      class="c47"
                    >
                      Cancel
                    </span>
                  </button>
                </div>
                <div
                  class="c44"
                >
                  <button
                    aria-disabled="false"
                    class="c45 c44 c15 c48"
                    type="button"
                  >
                    <span
                      class="c47"
                    >
                      Replace media
                    </span>
                  </button>
                  <span
                    class="c2"
                  >
                    <input
                      accept="image/png"
                      aria-hidden="true"
                      name="file"
                      tabindex="-1"
                      type="file"
                    />
                  </span>
                  <button
                    aria-disabled="false"
                    class="c45 c44 c15 c49"
                    type="button"
                  >
                    <span
                      class="c47"
                    >
                      Finish
                    </span>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
